### PR TITLE
Developing & Subgraphs 

### PR DESCRIPTION
**Pull request by idalithb**
- Re-org & editing copy

### CODE REVIEW

File: website/pages/en/subgraphs.mdx [hunk 1]
```diff
@@ -34,8 +40,47 @@ To learn more about each subgraph component, check out [creating a subgraph](/de
...
+Once defined, a subgraph can be [deployed to Subgraph Studio](/deploying/deploying-a-subgraph-to-studio/). In Subgraph Studio, you can do the following:
+
+- Use its staging environment to index the deployed subgraph and make it available for review.
+- Verify that your subgraph doesn't have any indexing errors and works as expected.
+
+### Publish to the Network
+
+When you're happy with your subgraph, you can [publish it](/publishing/publishing-a-subgraph/) to The Graph Network.
+
+- This is an on-chain action, which registers the subgraph and makes it discoverable by Indexers.
```
Comment (benface) [inline website/pages/en/subgraphs.mdx:58]: Do we use "on-chain" or "onchain"? cc @bermchain

File: website/pages/en/subgraphs.mdx [hunk 2]
```diff
@@ -34,8 +40,47 @@ To learn more about each subgraph component, check out [creating a subgraph](/de
...
+- Published subgraphs have associated metadata, which provides other network participants with useful context and information.
+
+### Add Curation Signal for Indexing
+
+Published subgraphs are unlikely to be picked up by Indexers without curation signal. To encourage indexing you should add signal to your subgraph. Learn more about signaling and [curating](/network/curating/) on The Graph.
+
+#### What is signal?
+
+- Signal is locked GRT associated with a given subgraph. It indicates to Indexers that a given subgraph will receive query volume and it contributes to the indexing rewards available for processing it.
+- Third party Curators may also signal on a given subgraph, if they deem the subgraph likely to drive query volume.
```
Comment (benface) [inline website/pages/en/subgraphs.mdx:69]:
```suggestion
- Third-party Curators may also signal on a given subgraph, if they deem the subgraph likely to drive query volume.
```

File: website/pages/en/subgraphs.mdx (hunk 1 above)
Comment (idalithb) [reply website/pages/en/subgraphs.mdx:58]: @benface I'll ask and edit

**Review by benface — APPROVED** [review]

File: website/pages/en/subgraphs.mdx (hunk 1 above)
Comment (idalithb) [reply website/pages/en/subgraphs.mdx:58]: on-chain more common  in web3 and typically used in our docs

**Pull request merged**